py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -138,6 +139,19 @@ def compile(
     if debug:
         set_log_level(logger.parent, logging.DEBUG)
 
+    if "truncate_long_and_double" in kwargs.keys():
+        if truncate_double is not _defaults.TRUNCATE_DOUBLE:
+            raise ValueError(
+                'Provided configuration for "truncate_double" and deprecated API "truncate_long_and_double", please only use "truncate_double"'
+            )
+        else:
+            truncate_double = kwargs["truncate_long_and_double"]
+            warnings.warn(
+                'Compiler option "truncate_long_and_double" is deprecated in favor of "truncate_double" as int64 is now natively supported, this option will be removed in the next version',
+                DeprecationWarning,
+                stacklevel=2,
+            )
```

Comment:
Any reason for not using `logger` here?

Reply:
This is apparently the recommended way to handle deprecation warnings, iirc I configured the logger to pull these messages in in an earlier PR